inc/com/centreon/engine/service.hh

```diff
@@ -136,6 +136,22 @@ class                           service : public notifier {
   void                          set_last_hard_state(enum service_state last_hard_state);
   enum service_state            get_initial_state() const;
   void                          set_initial_state(enum service_state current_state);
+  uint64_t                      get_flapping_comment_id(void) const;
+  void                          set_flapping_comment_id(uint64_t comment_id);
```

Comment:
Would it not be better to move this method in notifier ?
Do not we have the same one in host?

Reply:
done